tracer/test/Datadog.Trace.Tests/Logging/SerilogLogProviderTests.cs

```diff
@@ -29,76 +28,31 @@ public SerilogLogProviderTests()
                 .CreateLogger();
             _logEvents = new List<LogEvent>();
 
-            _logProvider = new CustomSerilogLogProvider();
+            _logProvider = new NoOpSerilogLogProvider();
             LogProvider.SetCurrentLogProvider(_logProvider);
             _logger = new LoggerExecutionWrapper(_logProvider.GetLogger("Test"));
         }
 
         [Fact]
-        public void LogsInjectionEnabledAddsParentCorrelationIdentifiers()
+        public void LogsInjectionEnabledDoesNotAddCorrelationIdentifiers()
```

Comment:
Perhaps this could be reduced even further by making this a `[Theory]` and test against `true` and `false`, just to ensure we're running the same code in both scenarios

Reply:
Done in 109a6dcb09d1bb4d548fef0f7e0bee305eaf8e1d